pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -191,6 +195,9 @@ rustPlatform.buildRustPackage rec {
     ZED_UPDATE_EXPLANATION = "zed has been installed using nix. Auto-updates have thus been disabled.";
```

Comment:
```suggestion
    ZED_UPDATE_EXPLANATION = "Zed has been installed using nix. Auto-updates have thus been disabled.";
```

Reply:
Unrelated to this PR, but I agree that looks nicer, will add it as a separate commit :)

Reply:
Yeah, hence not an actual review :D  It just caught my eye on the above screenshots